src/ResourceManager/EventHub/Commands.EventHub/ChangeLog.md

```diff
@@ -19,6 +19,8 @@
 -->
 ## Current Release
 * Updated help files to include full parameter types and correct input/output types.
+* Updated piping for InputObject and ResourceId in remove cmdlets
+* fixed few issues
```

Comment:
@v-Ajnava if there are any GitHub issues you could link to, would you please do that as a sub-bullet? If not, would you mind providing more information to the issues fixed

Reply:
removed the bullet * fixed few issues. in the previous release we have fixed the issues.